specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/quota.json

```diff
@@ -0,0 +1,1296 @@
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "tags": [
+          "Operation"
+        ],
+        "responses": {
+          "200": {
+            "description": "List all GET operations.",
+            "schema": {
+              "$ref": "#/definitions/OperationList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "OperationList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OperationResponse"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next page of items."
+        }
+      }
+    },
+    "OperationResponse": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplay"
+        },
+        "origin": {
+          "type": "string"
+        }
+      }
+    },
+    "OperationDisplay": {
+      "type": "object",
+      "properties": {
+        "provider": {
+          "type": "string",
+          "description": "Provider name."
+        },
+        "resource": {
+          "type": "string",
+          "description": "Resource name."
+        },
+        "operation": {
+          "type": "string",
+          "description": "Operation name."
+        },
+        "description": {
+          "type": "string",
+          "description": "Operation description."
+        }
+      }
+    },
+    "CommonResourceProperties": {
+      "type": "object",
+      "description": "Resource properties.",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource type. \"Microsoft.Quota/quotaLimits\""
+        }
+      }
+    },
+    "CurrentQuotaLimit": {
+      "type": "object",
+      "description": "Quota limit.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "quotaInformation": {
+          "description": "Quota information details.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CurrentQuotaLimitBase"
+        },
+        "properties": {
+          "description": "Additional properties for the quota request status for the specified resource.",
+          "$ref": "#/definitions/QuotaRequestStatusDetails",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
```

Comment:
It doesn't look like it is used anywhere. If so, please remove it.

Reply:
I will remove it.